test/openjd/cli/test_check_command.py

```diff
@@ -23,14 +23,12 @@ def test_do_check_file_success(tempfile_extension: str, doc_serializer: Callable
     """
     Execution should succeed given a correct filepath and JSON/YAML body
     """
-    temp_template = None
-
     with tempfile.NamedTemporaryFile(
         mode="w+t", suffix=tempfile_extension, encoding="utf8", delete=False
     ) as temp_template:
         doc_serializer(MOCK_TEMPLATE, temp_template.file)
 
-    mock_args = Namespace(path=Path(temp_template.name), output="human-readable")
+        mock_args = Namespace(path=Path(temp_template.name), output="human-readable")
```

Comment:
intended functional change? It's now within the context manager? I see this happening in a few spots

Reply:
Yea this looks correct. resolving